modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasks.path

```diff
@@ -113,6 +113,26 @@
 	<td>//tr[contains(.,'${key_workflowTask}') and contains(.,'${key_workflowAssetTitle}')]/td[2]//a</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_INPUT</td>
+	<td>//input[contains(@name,'keywords')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_ICON</td>
+	<td>//button[contains(@class, 'btn btn-monospaced btn-unstyled')][2]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_RESULT_BAR</td>
+	<td>//span[contains(text(),'${key_resultBarMessage}')]</td>
+	<td></td>
+</tr>
```

Comment:
You can make the name of those paths more clear, you can use that same path to both areas (Assigned to me / Assigned to my roles)
Suggestion:

MY_WORKFLOW_TASKS_SEARCH_INPUT_FIELD
MY_WORKFLOW_TASKS_SEARCH_ICON
MY_WORKFLOW_TASKS_SEARCH_RESULT_BAR

Remember to change the names at the created macro (searchAssignedToMe)